dotCMS/src/main/java/com/dotmarketing/portlets/containers/action/EditContainerAction.java

```diff
@@ -559,12 +563,16 @@ public void _saveWebAsset(ActionRequest req, ActionResponse res,
 		identifier.setHostId(host.getIdentifier());
 		APILocator.getIdentifierAPI().save(identifier);
 
+
+		List<ContainerStructure> oldContainerStructures = APILocator.getContainerAPI().getContainerStructures(currentContainer);
+		List<ContainerStructure> csList = new LinkedList<>();
+
 		// saving the multiple structures
 		if(container.getMaxContentlets()>0) {
 			String structuresIdsStr = req.getParameter("structuresIds");
 
 			String[] structuresIds = structuresIdsStr.split("#");
-			List<ContainerStructure> csList = new LinkedList<ContainerStructure>();
+			List<String> unusedStructures = new ArrayList<>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "unusedStructures". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "unusedStructures" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)